webpack.config.js

```diff
@@ -32,7 +32,11 @@ module.exports = {
     ]),
     new webpack.optimize.UglifyJsPlugin({
       sourceMap: false
-    })
+    }),
```

Comment:
Add a plugins section to your webpack.config.js to let know Webpack that the global PIXI variable make reference to pixi.js module. [Webpack Section ](https://github.com/pixijs/pixi.js/wiki/v5-Migration-Guide#webpack)